tests/playwright/pageobjects/recentchanges.page.js

```diff
@@ -0,0 +1,23 @@
+const Page = require( './page' );
+
+class RecentChangesPage extends Page {
+	constructor() {
+		super();
+		this.changesList = '.mw-changeslist';
+		this.changeListTitles = '.mw-changeslist-title';
+	}
+
+	async getTitles() {
```

Comment:
Since this function returns only the latest title, it should probably be renamed to something like getLatestTitle().

Reply:
This has been changed